src/run-scenario.jl

```diff
@@ -0,0 +1,46 @@
+export run_scenario
+
+"""
+    run_scenario(input_folder, output_folder)
+
+Run the scenario in the given input_folder and save the results to the output_folder
+"""
+
+function run_scenario(input_folder::AbstractString, output_folder::AbstractString)
```

Comment:
To me it depends on how long it takes to save the output. Or whether you can look at the output without saving it. 
I put them in one because my current model takes a loooong time to run, so it's nice to have it automatically save the output, in case anything happens to the computer before you get back to it.

I agree with your solution to have a flag / optional argument of "save output" where the default is True, but we can turn off if we want.